portality/settings.py

```diff
@@ -900,6 +920,52 @@
     "private_source": "portality.lib.query_filters.private_source",
     "public_source": "portality.lib.query_filters.public_source",
 }
+# Exclude the fields that doesn't want to be searched by public queries
+# This is part of non_public_fields_validator.
+EXCLUDED_FIELDS = [
```

Comment:
Can we give this a more specific name, like `PUBLIC_QUERY_VALIDATOR__EXCLUDED_FIELDS`, just so it's clear what it is out of context

Reply:
updated